### ISSUE TITLE
Pretrained lookup_embedder doesn't work for reciprocal_relations_model

### ISSUE THREAD

**Issue by nluedema** [uma-pi1/kge#190]
There is a problem with using pretrained relation embeddings for the lookup_embedder when using the reciprocal_relations_model. Only the first half (the non-reciprocal relation embeddings) are used, while the second half is ignored.

The problem seems to be with the function `_intersect_ids_with_pretrained_embedder` in `kge_model.py`.
```
elif "relation_embedder" in self.configuration_key:
    self_ids = self.dataset.relation_ids()
    pretrained_ids = pretrained_embedder.dataset.relation_ids()
```
The above code block only gets the first half of the relation id's and ignores the second half that was created by the reciprocal_relations_model.

**Comment by rgemulla** [comment]
Thanks for reporting this bug! @AdrianKs: Will you have a look?

**Comment by AdrianKs** [comment]
yes, I will have a look

**Comment by AdrianKs** [comment]
fixed in #191
